sdk/identity/identity/src/credentials/defaultAzureCredential.ts

```diff
@@ -44,9 +44,9 @@ interface DefaultCredentialConstructor {
  * A shim around ManagedIdentityCredential that adapts it to accept
  * `DefaultAzureCredentialOptions`.
  *
- * @internal
+ *
```

Comment:
I don't see this in the API review file, did you mean to stop marking this as internal?

Reply:
@KarishmaGhiya make sure to do `rushx build`, it will generate the review/*api file. Add it to this pr s🌞

Reply:
(sorry for my previous response, I was confused)

@KarishmaGhiya , while we exported this class to re-use it from the new credential, it’s still not a public class, so it should have the `@internal` comment 🙂